client/server/api/integration/index.js

```diff
@@ -1,3 +1,5 @@
+/* eslint jest/expect-expect: ["error", { "assertFunctionNames": ["expect", "localRequest.*.expect"] }] */
+
```

Comment:
The changes in this file are renaming `maybeIt` with `test`, and apply the automated refactor to return a Promise instead of using the arg `done`, but GitHub is making it hard to visualize.